### DISCUSSION TITLE
Remove image field from user's type

### DISCUSSION THREAD

**Discussion by nklisch** [nextauthjs/next-auth#7004] [Help]
I am building my own Adapter and workfow to work with an existing project that doesn't have an image.
I am able to merge my own fields into the default user type, but I am not able to remove fields, such as the image.
I can just ignore it, but in a group project, this means we have a field we never use.
Is there anyway with typescript to omit this type when using the suggested declaration merging that is suggested?

**Comment by srj-github** [comment]
Hi,
Have you managed to remove the field?
Thanks,

**Comment by pedropalau** [comment]
Did you find the way to remove that field @nklisch?

**Comment by debpu06** [comment]
Pinging again @nklisch to see if you found a way to remove the image. Thanks!
